### PR TITLE
DRAFT PR - Ac docs pre cohort 4 revisions

### PR DESCRIPTION
**Pull request by AndrewB9lab**
This PR includes edits to various parts of the program resulting from the following:

* Review of the new Interchain Security module
* Language update to use "Acknowledgement" across the board, and other minor edits (cherry picked to #1382)
* Contextual update to replace "Tendermint" with "CometBFT" where appropriate (cherry picked to #1382)
* Correction of broken links and missing images
* Renaming and restructuring activities

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [X] Small language/grammar fixes
* [X] Small content fixes 
* [X] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

**Comment by AndrewB9lab** [comment]
NOTE: this PR can be ignored for the time being. @CitMC and I are performing QA on a branch from this that has merged in the most recent Master; likely this branch will be abandoned after QA is completed.

File: ida-customizations/ida-course/LPs/week-0/index.md [hunk 1]
```diff
@@ -21,7 +21,17 @@ If you do not need a blockchain refresher, feel free to start your journey with
...
+
+Working with Cosmos also means working with Golang. If you have not worked much with Go, do not worry.
+
+<HighlightBox type="info">
+
+As all blockchain ecosystems keep on growing and maturing, product stacks and terminologies evolve. Thus, please be aware that Cosmos is also known as **the Interchain**. The terms "Cosmos", "Cosmos Ecosystem", and "Interchain" are synonymous with each other, and the various tools available to developers ("the stack") can be referred to collectively as the **Interchain Stack**.
+
+</HighlightBox>
+
+This introduction to Golang is meant as a useful overview of Go and as such, a starting point for your journey in the Cosmos Ecosystem and developing with the Cosmos SDK.
```
Comment (xavierlepretre) [inline ida-customizations/ida-course/LPs/week-0/index.md:34]: @AndrewB9lab This is the only meaningful remaining difference in this PR. If you think it is not important, just close the PR.

Comment (AndrewB9lab) [reply ida-customizations/ida-course/LPs/week-0/index.md:34]: Yes, I think this is superseded by what we have now.

**Comment by AndrewB9lab** [comment]
Closing comment: all actions of substance were implemented during the QA process via a branch from this branch; other changes deemed superseded.

**Pull request closed without merging**